Orange/widgets/tests/test_stack.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+from Orange.data import Table
+from Orange.modelling import KNNLearner, TreeLearner
+from Orange.evaluation import CA, CrossValidation, MSE
+
+from orangecontrib.prototypes.stack import StackedFitter
```

Comment:
This still tries to import from prototypes.

Reply:
I thought I changed this. 😮 Fixed.